examples/advanced-project/src/__checks__/synthetics/02-business-critical.check.ts

```diff
@@ -0,0 +1,25 @@
+import * as path from 'path'
+import { BrowserCheck } from 'checkly/constructs'
+import { syntheticGroup } from '../utils/website-groups.check'
+
+// In this example, we configure two checks for our homepage in a single
```

Comment:
Nit: Generally, I'd say short is always better in docs. "In this example," can be omitted. 

So I'd go with:

> Two checks for the homepage in a single ...

I'd also step back from "our", "my", etc. in docs and examples. :)